src/ice/console.py

```diff
@@ -19,11 +19,12 @@
 import filesystem_helper
 import utils
 from persistence.backed_object import BackedObject
-from persistence.config_file_backing_store import ConfigFileBackingStore
-from emulator import Emulator
+# from persistence.config_file_backing_store import ConfigFileBackingStore
+# from emulator import Emulator
```

Comment:
Please don't leave commented imports. If they are unused then you can just delete them.